dataset_creation.ipynb

```diff
@@ -449,82 +3112,101 @@
    ]
   },
   {
-   "source": [
-    "### Save with Pickle"
-   ],
    "cell_type": "markdown",
-   "metadata": {}
+   "metadata": {},
+   "source": [
+    "### Dropping extra columns"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 40,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pickle\n",
-    "with open('movieDB.pickle', 'wb') as f:\n",
-    "    pickle.dump(movieDB, f)"
+    "#based on columns having more than 85% empty entries\n",
+    "movieDB = pd.DataFrame(movieDB)\n",
+    "movieDB.drop(['Japanese','Hepburn','Adaptation by','Animation by','Traditional','Simplified','Created by','Original work','Owned by'],axis=1,inplace=True)"
    ]
   },
   {
-   "source": [
-    "### Load from Pickle"
-   ],
    "cell_type": "markdown",
-   "metadata": {}
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
    "metadata": {},
-   "outputs": [],
    "source": [
-    "with open('movieDB.pickle', \"rb\") as f:\n",
-    "    movieDB = pickle.load(f)"
+    "### Save with Pickle"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 41,
    "metadata": {},
    "outputs": [],
    "source": [
-    "movieDB[0]"
+    "import pickle\n",
+    "with open('movieDB.pickle', 'wb') as f:\n",
+    "    pickle.dump(movieDB, f)"
    ]
   },
   {
-   "source": [
-    "## Save Data as CSV"
-   ],
    "cell_type": "markdown",
-   "metadata": {}
+   "metadata": {},
+   "source": [
+    "### Load from Pickle"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 42,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pandas as pd"
+    "with open('movieDB.pickle', \"rb\") as f:\n",
+    "    movieDB = pickle.load(f)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Save Data as CSV"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 44,
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = pd.DataFrame(movieDB)"
+    "df.to_csv('movieDB_new.csv')"
```

Comment:
You're still saving the old dataframe named df to csv here instead of the movieDB dataframe.